### DISCUSSION TITLE
Issue while display at frontend side.

### DISCUSSION THREAD

**Discussion by rdhathaliya** [GrapesJS/grapesjs#5902] [Q&A] [answered]
I have tried to do something like image 1 but on my frontend page its show like image 2 please help
![Screenshot 2024-05-23 145142](https://github.com/GrapesJS/grapesjs/assets/169676568/417b53f0-f5a2-4980-9bf5-abc85f46bae7)

![Screenshot 2024-05-23 145333](https://github.com/GrapesJS/grapesjs/assets/169676568/74443b4a-300c-4abe-acca-8dbb2ba38986)

**Comment by artf** [comment, marked as answer]
Please share a reproducible demo

**Reply by rdhathaliya** [reply]
Thanks @artf! I found the solution. I had forgotten to import tailwind.css. The components are created using [grapesjs-tailwind](https://unpkg.com/grapesjs-tailwind).
